### ISSUE TITLE
Sorting

### ISSUE THREAD

**Issue by jjnj023** [galadril/DomoticzForAndroid#119]
Currently the devices are sorted alphabetically. It would be a better option to get the same sorting as in Domoticz (Dromotica app does the same) or let the user choose which sorting option to use.

**Comment by RoadXY** [comment]
I agree on that alphabetical order isn't always the best for mobile use.
However the order of Domoticz isn't always the best for mobile.
Custom ordering will bring a lot of other complications like what to do when an device is added or deleted from the server.

Are the widgets an solution for this?
Did you use the filter option (the 4 horizontal lines)? Is that usable to solve this?

**Comment by jjnj023** [comment]
I do use the widgets now. But I think a more elegant solution will be either the same sorting or indeed a custom sorting.
I don't see many obstacles to overcome. When new devices are added they can be added at the bottom of the list for example. 
But why not give the user the option to either choose from alphabetically or server sorting?

**Comment by RoadXY** [comment]
Because more options often lead to confusion ;-)
I'll discuss this with Mark.

**Comment by jjnj023** [comment]
People able to install Domoticz should also understand these options i think ;-) (or explain to their wives hehe)

**Comment by RoadXY** [comment]
Indeed, a V2 issue.

**Comment by RoadXY** [comment]
Thanks
